_documentation/HowToUse.md

```diff
@@ -4,336 +4,207 @@ permalink: documentation/HowToUse.html
-    <li>Choose one from <a href="#check-online-file">the Internet</a> by selecting the “check online files” tab, or</li>
-    <li>Use the test files uploaded to the server by selecting the “check server files” tab</li>
-  </ul>
+* [Overview](#overview)
+* [Using MediaConchOnline and MediaConch GUI](#using-mediaconchonline-and-mediaconch-gui)
+* [Using MediaConch CLI](#using-mediaconch-cli)
 
-  <li>Click “Check Files”</li>
-  <li>Once checked, the results will appear below and you can see the reports for each section by clicking on the “eyecon” <img src="../images/eyecon.png" alt="eyecon" /> (3rd screenshot) or download them by clicking the <img src="../images/downArrow.png" alt="down arrow" /></li>
-</ol>
+# Overview
 
-Checker:
+## Installation Guide
 
-![MediaConchOnlineGuiHome](../images/MCOnlineGUI1Home.png)
-![MediaConchOnlineGuiChecked](../images/MCOnlineGUI2Checked.png)
+MediaConch is available for use as a command line interface (CLI), a graphical user interface (GUI) and a web user interface (WebUI). See [our installation guide](https://mediaarea.net/MediaConch/installation.html) how to install with the interface of your choosing.
 
-Policy Report:
+## How to use MediaConch
 
-![MediaConchOnlineGUIPolicyReport](../images/MCOnlineGUI3PolicyReport.png)
+MediaConch works with virtually any file format, but is created to work specifically with Matroska-wrapped FFV1 and LPCM encoded video files. For these files, validation is performed according to the specifications of each of these formats. If, for example, an AVI-wrapped FFV1 and LPCM encoded video file is used, thorough file validation will be performed on the FFV1 and LPCM portions and the AVI file will be minimally checked for validation and approved if these validations pass. MediaConch works with sister projects [veraPDF](http://verapdf.org/) and [DPF Manager](http://dpfmanager.org/) to perform thorough validation on PDF and TIFF file formats.
 
-Policies:
+\*Sponsorship is available for validation-checking for other file formats, when applicable.
 
-![MediaConchOnlineGUIPolicies](../images/MCOnlineGUI4Policies.png)
+## Checking files
 
-###MediaConch currently consists of three main sections: “Checker,” “Policies,” and “Display.”
+Files from various sources can be checked with MediaConch, regardless of the chosen interface. A preservationist may check local files, online files, or local folders. "Check local file" allows you to select a file or files from a local computer. "Check online file" allows you to select a file using a URL path. Please note that this feature supports the following application protocols: HTTP/HTTPS/FTP/FTPS. "Check local folder" allows you to select a folder of files from a local computer or volume. For any of these selections, a preservationist may choose from either an existing policy and display in MediaConch or from an imported XSL/Schematron policy file.
```

Comment:
Oh schematron? IMHO if schematron is not yet deprecated, it should be and we should reduce references to it.